PirateGame/Assets/Scripts/UserShipAgent.cs

```diff
@@ -94,6 +90,7 @@ void Update ()
 			//underneath the messages for maxed upgrades and no loot.
 
 		}
+		//TODO: should be able to move in waiting state so you can move around the island, not just get stopped on one side
```

Comment:
Ship movements are very broad, they wouldn't be able to maneuver well in the space of a small island and it wouldn't make a difference to their looting.  Also, last week everybody agreed on having the ship stop and then resume turned around when done.